src/dotnet-svcutil/lib/src/DebugLogger.cs

```diff
@@ -101,10 +125,31 @@ void NuGet.Common.ILogger.LogInformationSummary(string data)
             LogMessage(data, LogTag.LogMessage);
         }
 
+        void NuGet.Common.ILogger.Log(NuGet.Common.ILogMessage message)
```

Comment:
Why is this change needed for updating the Newtonsoft packages?

Reply:
> Why is this change needed for updating the Newtonsoft packages?

The interface ILogger in Higher version of NuGet.Common package has defined more methods, we should  have the implementation in the inherited class